ros2pkg/ros2pkg/verb/create.py

```diff
@@ -143,6 +143,11 @@ def main(self, *, args):
         if args.cpp_library_name:
             print('cpp_library_name:', args.cpp_library_name)
 
+        package_path = os.path.join(args.destination_directory, package.name)
```

Comment:
This is definitely better.  However, I think it would be better to have this error occur before the `print`s above, just so the user doesn't get confused by some output followed by an error.

Reply:
Done: 48a1941cea62d5b55c255a3de809b96a2d03e53b